src/lamplib/src/genny/toolchain.py

```diff
@@ -16,7 +16,7 @@
 # Define complex operations as private methods on the module to keep the
 # public Context object clean.
 def _create_compile_environment(
-    triplet_os: str, toolchain_dir: str, system_env: Optional[dict] = None
+    triplet_os: str, toolchain_dir: str, arch: str, system_env: Optional[dict] = None
```

Comment:
pedantry: make `arch` and `triplet_os` into one tuple or at least consecutive args. Only worth it if your IDE supports that kind of automated refactor.

Reply:
I'd prefer to leave this one for now. I agree it would be better, but also increases the changes of something breaking and I have enough of those right now :)